### ISSUE TITLE
Slippage

### ISSUE THREAD

**Issue by avocardow** [AaronCQL/bluna-bot#4]
I know that you said slippage won't be affected by the min-gain and max-loss percentages. But sometimes unfortunately when those percentages are really tight, the slippage can still cause unprofitable trades. I know this can be fixed by accounting for this in the slippage as well as the min-gain and max-loss settings, but I was just wondering if it would be possible for you to make setting a specific slippage % be optional, but relying more on whether we still are within the min-gain and max-loss settings.

Basically, if a swap is attempted, before executing we make sure that the slippage hasn't exceeded the min-gain or max-loss, and using those as whether or not the swap gets cancelled or completed.

Thank you so much
